config/fcos/v1_6_exp/validate.go

```diff
@@ -51,3 +51,19 @@ func (user GrubUser) Validate(c path.ContextPath) (r report.Report) {
 	}
 	return
 }
+
+func (c Config) Validate() (r report.Report) {
+	for _, disk := range c.Storage.Disks {
+		if disk.Device == "/dev/disk/by-id/coreos-boot-disk" {
```

Comment:
Hmm, maybe we could join the two if blocks? 
If the device is not root, and wipe table is set to false, lets validate.

Reply:
no problem! , my thinking was you could check if its the root disk and if it is then skip the rest of the test.